Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.2.3
+
+RUN apt-get update -qq && \
+  apt-get install -y build-essential && \
+  apt-get install -y libpq-dev && \
+  apt-get install -y libxml2-dev libxslt1-dev && \
+  apt-get install -y libqt4-webkit libqt4-dev xvfb && \
```

Comment:
I'm not sure if capybara was updated before this PR was originally put in, but during testing, this doesn't provide a working image; capybara would fail to compile.  I suspect the above are slightly older than what [thoughtbot may support](https://github.com/thoughtbot/capybara-webkit/wiki/Installing-Qt-and-compiling-capybara-webkit#debian-testing-unstable-stable-wheezy-ubuntu)